### PR TITLE
:bug: (report) do not include offbudget transfers in cash-flow card expenses

### LINKED ISSUE #2237: [Bug]: Mismatch Between Cash Flow Card and Cash Flow Drill-Down Report
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

Using the [savings workflow recommended in the "Recommended workflow for new users"](https://actualbudget.com/docs/budgeting/how-it-works/), when money is transferred into a savings account and using the savings transaction category, **savings is correctly shown as a transfer, distinct from expenses, in the cash flow drill down report** (https://<domain>/reports/cash-flow).

However, when viewing the **Cash Flow Card (https://<domain>/reports), the transfer (and thus savings) workflow is bucketed with expenses, thus reporting an incorrectly inflated expense flow**.

**tl;dr:** Transfers are classified as expenses in the Cash Flow Card which should not be the case.

### Why this is a problem

This mismatch is not only detrimental from a reporting perspective, but also from a usability/Actual budget mindset where "The purpose of a budget is to answer simple questions: how much money did I save last month?" is blurred because savings are now treated as expenses.

### What should happen

Since many/most users use the cash flow card as a quick gauge check to see if they're living within their means (i.e. positive, or at least neutral cashflow), this mismatch should be fixed where the Cash Flow Card filters only on expenses and does not include expenses, just as its drill-down report does. (This should also be fixed in the sum of income + expenses in the top right of the card).

### What error did you receive?

_No response_

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Microsoft Edge

### Operating System

Windows 11

### PR DESCRIPTION
**Pull request by MatissJanis**
Closes #2237

Do not include off-budget transfers in the "expenses" in the cash flow card
Screenshot 2024-03-21 at 18 58 09

Demo budget file: 
[2024-03-21-_test-budget.zip](https://github.com/actualbudget/actual/files/14705805/2024-03-21-_test-budget.zip)

### CODE REVIEW

**Review by youngcw — APPROVED** [review]

**Comment by rhorno** [comment]
Couldn't one achieve the same by just filtering out transfers in the widget before this PR? 
And by that, not breaking the former functionality that enabled users that transfer(expense) to off-budget debt accounts to correctly show their cash flow.
